test.sh

```diff
@@ -3,5 +3,14 @@
 set -e
 
 go build -o ./dist/klone_test .
+klone_binary=$(realpath ./dist/klone_test)
 
-cd ./example && ../dist/klone_test sync
+pushd ./example
+"$klone_binary" sync
+popd
+
+temp_dir=$(mktemp -d)
+trap '{ rm -rf "$temp_dir"; echo "> Deleted temp dir $temp_dir"; }' EXIT
+pushd "$temp_dir"
+"$klone_binary" init
+popd
```

Comment:
suggestion: maybe a separate PR, but it'd be worth adding something like this with `klone add a b https://github.com/cert-manager/community.git main logo` on an empty repo as well!